### ISSUE TITLE
pset naming issues

### ISSUE THREAD

**Issue by tehn** [monome/norns#1086]
- When you save to slot 1 you can only name the PSET with the name of the script. For eg. with Moln, whatever I write into the field the preset will always just be called “moln”.
- locations other than 1 seem to work at first. But when you try to overwrite the preset, you’ll get presented with the default name again.

**Comment by papernoise** [comment]
So far it seems to me that this happens only with some scripts. Molly the Poly for eg. seems to work fine. I can test through more scripts to see if there's a pattern.

**Comment by papernoise** [comment]
ok I went through a couple of scripts.

I tried thiese: Awake, Molly the Poly, Bounds, Cheat Codes, Circles, Moln
giving presets a custom name works the first time, but when overwriting them I get the default name again.

Last time I tried things were a bit more random. For eg. Moln wouldn't let me enter a custom preset name at all (on any slot), and Molly the Poly was working as expected.

Tried Otis then, and it would have the same issue as I previously had with Moln. It would not let me enter a custom preset name on the first slot, always just showing "moln".
The name would still revert to the standard "otis" when overwriting an existing preset.

For now this seems to be happen a bit randomly,  I need to investigate this further when I have a moment.
